content/administration/install/deploy.rst

```diff
@@ -388,20 +388,109 @@ The solutions to support livechat/motifications in a WSGI application are:
   with ``/longpolling/`` to
   :option:`the longpolling port <odoo-bin --longpolling-port>`.
 
+.. _streaming:
+
+Serving Static Files and Attachments
+====================================
+
+For development convenience, Odoo directly serves all static files and attachments in its modules.
+This may not be ideal when it comes to performances, and static files should generally be served by
+a static HTTP server.
+
 Serving Static Files
-====================
+--------------------
+
+Odoo static files live in each module's ``static/`` folder, so static files can be served by
+intercepting all requests to :samp:`/{MODULE}/static/{FILE}`, and looking up the right module
+(and file) in the various addons paths.
+
+Say Odoo has been installed via the **debian packages** for community and enterprise, the addons
+paths is ``/usr/lib/python3/dist-packages/odoo/addons``. Using the above (https) NGINX
+configuration, the following location block should be added to serve static files via NGINX.
+
+.. code-block:: nginx
+
+    location @odoo {
+        # copy-paste the content of the / location block
+    }
+
+    # Serve static files right away
+    location ~ ^/[^/]+/static/.+$ {
+        root /usr/lib/python3/dist-packages/odoo/addons;
+        try_files $uri @odoo;
+        expires 24h;
+    }
+
+Say Odoo has been installed via the **source**, the two git repositories for community and
+enterprise has been cloned in ``/opt/odoo`` and ``/opt/odoo-enterprise`` respectively. The addons
+paths is ``/opt/odoo/odoo,/opt/odoo/addons,/opt/odoo-enterprise``. Using the above (https) NGINX
+configuration, the following location block should be added to serve static files via NGINX.
+
 
-For development convenience, Odoo directly serves all static files in its
-modules. This may not be ideal when it comes to performances, and static
-files should generally be served by a static HTTP server.
+.. code-block:: nginx
+
+    location @odoo {
+        # copy-paste the content of the / location block
+    }
+
+    # Serve static files right away
+    location ~ ^/[^/]+/static/.+$ {
+        try_files /base$uri /addons$uri /enterprise$uri @odoo$uri;
+        expires 24h;
+    }
 
-Odoo static files live in each module's ``static/`` folder, so static files
-can be served by intercepting all requests to :samp:`/{MODULE}/static/{FILE}`,
-and looking up the right module (and file) in the various addons paths.
+    location /base {
+        internal;
+        alias /opt/odoo/odoo/addons;
+    }
+
+    location /addons {
+        internal;
+        alias /opt/odoo/addons;
+    }
+
+    location /enterprise {
+        internal;
+        alias /opt/odoo-enterprise;
+    }
+
+.. warning::
+
+  The actual NGINX configuration you need is highly dependant on your own installation. The two
```

Comment:
```suggestion
  The actual NGINX configuration you need is highly dependent on your own installation. The two
```
Also, the blank line between the warning admonition and its content isn't necessary.